drizzlepac/sky.py

```diff
@@ -39,53 +40,438 @@
 #this is the user access function
 def sky(input=None,outExt=None,configObj=None, group=None, editpars=False, **inputDict):
     """
-    Perform sky subtraction on input list of images
+    Function for computing and subtracting (or equalizing/matching) the backgroud
+    in input images. The algorithm for sky subtraction can be selected through
+    the ``skymethod`` parameter. This function will update the ``MDRIZSKY`` keyword
+    in the headers of the input files.
+
+    Sky subtraction is generally recommended for optimal flagging and removal of
+    CR's when the sky background is more than a few electrons.  However, some
+    science applications may require the sky to not be removed, allowing for the
+    final drizzle step to be performed with no sky subtraction. If you turn off
+    sky subtraction, you should also set drizzle.pixfrac to 1, otherwise
+    variations in sky between images will add noise to your data.
+
+    In addition to the "pure" sky computation, this task can be used for sky
+    "equalization", that is, it can match sky values in the images
+    that are part of a mosaic.
+
+    For cameras with multiple detectors (such as ACS/WFC, WFPC2, or WFC3),
+    the sky values in each exposure are first measured separately for
+    the different detectors. These different values are then compared,
+    and the lowest measured sky value is used as the estimate for all of the
+    detectors for that exposure. This is based on the premise that for large
+    extended or bright targets, the pixel intensity distribution in one or more
+    of the detectors may be significantly skewed toward the bright end by the
+    target itself, thereby overestimating the sky on that detector. If the other
+    detector is less affected by such a target, then its sky value  will be lower,
+    and can therefore also be substituted as the sky value for the detector
+    with the bright source. The input file's primary headers is updated with the
+    computed sky value.
+
+    For more information on the science applications of the sky task,
+    see the `DrizzlePac Handbook: <http://drizzlepac.stsci.edu>`_.
+
 
     Parameters
     ----------
-    input : str or list of str
-        a python list of image filenames, or just a single filename
-    configObj : configObject
-        an instance of configObject
-    inputDict : dict, optional
-        an optional list of parameters specified by the user
-    outExt : str
+
+    input : str or list of str (Default = None)
+        A Python list of image filenames, or just a single filename.
+
+    outExt : str (Default = None)
         The extension of the output image. If the output already exists
-        then the input image is overwritten
+        then the input image is overwritten.
+
+    configObj : configObject (Default = None)
+        An instance of ``configObject``
```

Comment:
It may be worth adding a bit more about this parameter. As I am not familiar with it I wonder what it should include. Perhaps add a link to a snippet elsewhere describing the contents?